### PR TITLE
Adding automated tasks section to documentation

### PR DESCRIPTION
**Pull request by daniel-samson**
This pull request is predicated on https://github.com/salesagility/SuiteCRM/pull/5735/ being merged into the project.

This change adds how to use robo with suitecrm.

### CODE REVIEW

File: content/developer/AutomatedTasks/Sass.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+Title: "Building Sass"
+---
+
+== SuiteP Theme
+
+SuiteCRM provies an automated to task to build suitep colors
```
Comment (CBeechey) [inline content/developer/AutomatedTasks/Sass.adoc]: typo - should be provides and SuiteP

File: content/developer/AutomatedTasks/TestingEnvironment.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+Title: "Configure Testing Environment"
+---
+
+== Testing Environment
+
+SuiteCRM provies an automated to task to assist with the configuration of the testing environment.
```
Comment (CBeechey) [inline content/developer/AutomatedTasks/TestingEnvironment.adoc]: typo - should be provides

File: content/developer/AutomatedTasks/Sass.adoc (hunk 1 above)
Comment (daniel-samson) [reply content/developer/AutomatedTasks/Sass.adoc]: fixed

File: content/developer/AutomatedTasks/TestingEnvironment.adoc (hunk 2 above)
Comment (daniel-samson) [reply content/developer/AutomatedTasks/TestingEnvironment.adoc]: fixed

**Pull request merged**